src/ukify/ukify.py

```diff
@@ -858,6 +893,55 @@ def generate_keys(opts):
             pub_key.write_bytes(pub_key_pem)
 
 
+def pe_read_single_section(section, opts, name, text):
+    data = section.get_data(ignore_padding=True)
+    size = section.Misc_VirtualSize
+    h = sha256(data).hexdigest()
+    data_printed = None
+
+    tmp = {
+        'size' : size,
+        'sha256' : h,
+    }
+
+    if text:
+        try:
+            data_printed = data.decode()
+        except UnicodeDecodeError:
+            print(f"Warning! {name} cannot be read. Not a valid UTF8-encoded text")
+            data_printed = None
+        finally:
+            tmp['text'] = data_printed
+
+    if opts.json:
+        out_str = json.dumps(tmp)
+    else:
+        out_str = f"{name}:\nsize: {size} bytes\nsha256: {h}\n"
+        if text:
+            out_str += f"text:\n{data_printed}\n"
+        print(out_str)
```

Comment:
If you want to print things out, why not just print them?
```python
print("f"{name}:\nsize: {size} bytes\nsha256: {h}")
if text:
  print(…)
```
?

Reply:
Because below there is 
```
    if name in Section.files_to_write:
        Section.files_to_write[name].write_text(out_str)
```
that uses out_str.

Reply:
Oh, so this both prints to stdout and to the file. I would expect it to not print anything to the terminal if the output options are given.

Reply:
No, it prints the specific section to the file, and the whole thing to stdout.